src/Parser.elm

```diff
@@ -0,0 +1,45 @@
+module Parser exposing (..)
+
+{- Sample:
+   Compressed Twinkling Cobaltite    4397
+   Compressed Glowing Carnotite      2603
+   Glowing Cinnabar                  1918
+   Glistening Coesite                2740
+-}
+
+
+type alias OreEntry =
+    { name : String
+    , volume : Int
+    }
+
+
+parseOreInputLine : String -> Maybe OreEntry
+parseOreInputLine line =
```

Comment:
This function would be a great candidate for unit tests -- https://package.elm-lang.org/packages/elm-explorations/test/latest/